### PR TITLE
Load static assets from static dir

### PR DESCRIPTION
**Pull request by mrskug**

### CODE REVIEW

File: apps/mosaico/src/Main.purs [hunk 1]
```diff
@@ -203,7 +206,9 @@ renderArticle env@{ htmlTemplate } uuid article mostReadArticles = do
       in notFound env maybeMostRead { params: {path: foldMap (List.fromFoldable <<< (_ `cons` ["artikel"])) uuid} }
 
 assets :: { params :: { path :: List String } } -> Aff (Either Failure File)
-assets { params: { path } } = Handlers.directory "dist/client" path
+assets { params: { path } } = do
+  Console.log $ "Fetching asset with internal route: " <> (intercalate "" path)
```
Comment (vapaj) [inline apps/mosaico/src/Main.purs:210]: We can probably remove the logging

File: ci/app-servers/mosaico.dhall [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    , domains = [ "mosaico.app.ksfmedia.fi/*" ]
+    , handlers =
+      [ Handler::{ url = Some "/assets", static_dir = Some "dist/client" } ]
+    , entrypoint =
+        ''
+          node -e "require('./output/Main/index').main()"
+        ''
+    , env = toMap
+        { PERSONA_URL = "https://persona.staging.ksfmedia.fi/v1"
+        , LETTERA_URL = "https://lettera.staging.ksfmedia.fi/v3"
```
Comment (vapaj) [inline ci/app-servers/mosaico.dhall:23]: For mosaico, stage lettera should be `v4beta`

Comment (mrskug) [reply ci/app-servers/mosaico.dhall:23]: this is the prod env, stage is still configured with the `app.dev.yaml`

Comment (vapaj) [reply ci/app-servers/mosaico.dhall:23]: I see, then this could use prod lettera and persona

Comment (mrskug) [reply ci/app-servers/mosaico.dhall:23]: ah, yes :)

**Review by vapaj — APPROVED** [review]

**Pull request merged**